### PR TITLE
CSS docs

### PR DESCRIPTION
**Pull request by ellisonbg**
This PR improves the CSS documentation based on the new CSS variable usage.

### CODE REVIEW

File: src/default-theme/variables.css [hunk 1]
```diff
@@ -3,47 +3,88 @@
 | Distributed under the terms of the Modified BSD License.
 |----------------------------------------------------------------------------*/
 
+/*
+The following CSS variables define the main, public API for styling JupyterLab.
+These variables should be used by all plugins whereever possible. In other
```
Comment (blink1073) [inline src/default-theme/variables.css]: Wherever*

File: src/default-theme/variables.css [hunk 2]
```diff
@@ -3,47 +3,88 @@
...
+words, plugins should not define custom colors, sizes, etc unless absolutely
+necessary. This enables users to change the visual theme of JupyterLab
+by changing these variable.
```
Comment (blink1073) [inline src/default-theme/variables.css]: variables*

File: src/default-theme/variables.css [hunk 3]
```diff
@@ -3,47 +3,88 @@
...
+
+Many variables appear in an ordered sequence (0,1,2,3). These sequences
+are designed to work well together, so for example, --jp-border-color1 should
```
Comment (blink1073) [inline src/default-theme/variables.css]: Add backticks around the variable names

File: src/default-theme/variables.css [hunk 4]
```diff
@@ -3,47 +3,88 @@
...
+be used with --jp-layout-color1. The numbers have the following meanings:
+
+* 0: super-primary, reserved for special emphasis
+* 1: primary, most important under normal situations
+* 2: secondary, next most important under normal situations
+* 3: tertiary, next most important under normal situations
+
+Throughout JupyterLab, we are mostly following principles from Google's
+Material Design when selecting colors. We are not, however, following 
+all of MD as it not optimized for dense, information rich UIs.
```
Comment (blink1073) [inline src/default-theme/variables.css]: it is*

File: src/default-theme/variables.css [hunk 5]
```diff
@@ -3,47 +3,88 @@
...
   --jp-content-font-size: 14px;
   --jp-content-font-color0: black;
   --jp-content-font-color1: black;
   --jp-content-font-color2: var(--md-grey-700);
   --jp-content-font-color3: var(--md-grey-500);
 
-  /* Layout */
+  /* Layout
+  
+  The following are the main layout colors use in JupyterLab. In a light`
```
Comment (blink1073) [inline src/default-theme/variables.css]: errant backtick

File: tutorial/css.md [hunk 6]
```diff
@@ -3,24 +3,73 @@
...
+* Where possible, private variables should be defined and scoped under an 
+  appropriate selector.
+
+### CSS variable usage
+
+JupyterLab includes a default set of CSS variables in the file:
+
+`src/default-theme/variables.css`
+
+To ensure consistent design in JupyterLAb, all built-in and third party
```
Comment (blink1073) [inline tutorial/css.md]: JupyterLab*

File: src/default-theme/variables.css (hunk 1 above)
Comment (ellisonbg) [inline src/default-theme/variables.css]: done

File: src/default-theme/variables.css (hunk 2 above)
Comment (ellisonbg) [inline src/default-theme/variables.css]: done

File: src/default-theme/variables.css (hunk 4 above)
Comment (ellisonbg) [inline src/default-theme/variables.css]: done

File: src/default-theme/variables.css (hunk 3 above)
Comment (ellisonbg) [inline src/default-theme/variables.css]: done

File: src/default-theme/variables.css (hunk 5 above)
Comment (ellisonbg) [inline src/default-theme/variables.css]: done

File: tutorial/css.md (hunk 6 above)
Comment (ellisonbg) [inline tutorial/css.md]: done

**Comment by ellisonbg** [comment]
All review comments addressed...

**Comment by blink1073** [comment]
👍

**Pull request merged**